kong/plugins/datadog/schema.lua

```diff
@@ -79,9 +80,10 @@ return {
           { service_name_tag = { type = "string", default = "name" }, },
           { status_tag = { type = "string", default = "status" }, },
           { consumer_tag = { type = "string", default = "consumer" }, },
-          { retry_count = { type = "integer", required = true, default = 10 }, },
-          { queue_size = { type = "integer", required = true, default = 1 }, },
-          { flush_timeout = { type = "number", required = true, default = 2 }, },
+          { retry_count = { type = "integer" }, },
+          { queue_size = { type = "integer" }, },
+          { flush_timeout = { type = "number" }, },
```

Comment:
Can we move these to aliases so that they just modify the `queue` record?

Reply:
They have semantics different from the old parameters and we want users to switch over to the new parameters instead.